message-me/app/models/message.rb

```diff
@@ -1,4 +1,5 @@
 class Message < ApplicationRecord
   belongs_to :user
   validates :body, presence: true
+  scope :custom_display, -> { order(:created_at).last(20) }
```

Comment:
1. Isn't the default already ordered by `created_at`?
2. Naming sucks. Find a better name.
3. If you going to remove the `order` here, does it even make sense to create a scope just for this?